frontend/src/components/Account/common/Tabs/Tabs.js

```diff
@@ -0,0 +1,75 @@
+import React, { useState, useEffect } from "react";
+import PropTypes from "prop-types";
+import { useHistory, useLocation } from "react-router-dom";
+import Tab from "../Tab/Tab";
+import "./Tabs.scss";
+
+export const Tabs = ({ tabs, defaultTab }) => {
+  const history = useHistory();
+  const location = useLocation();
+  // selected tab by default will be the first tab
+  const [selectedTab, setSelectedTab] = useState(defaultTab);
+  useEffect(() => {
+    const openedTab = tabs.find(
+      tab => location.pathname.indexOf(tab.url) !== -1
+    );
+    if (openedTab) {
+      setSelectedTab(openedTab.id);
+    }
+    const url = openedTab
+      ? location.pathname + location.search
+      : `${tabs[selectedTab].url}`;
```

Comment:
Is this just interpolating a string into itslef?

Reply:
removed code after a refactoring related to tabs